lib/decodeCalldata.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'earljs'
+
+import { decodeCalldata } from './decodeCalldata'
+import { parseAbi } from './parseAbi'
+
+describe(decodeCalldata.name, () => {
+  it('handles expected case', () => {
+    const iface = parseAbi('function transferFrom(address,address,uint256),')!
+    expect(iface).toBeDefined()
+    expect(
+      decodeCalldata(
+        iface,
+        '0x23b872dd0000000000000000000000008ba1f109551bd432803012645ac136ddd64dba72000000000000000000000000ab7c8803962c0f2f5bbbe3fa8bf41cd82aa1923c0000000000000000000000000000000000000000000000000de0b6b3a7640000',
+      ),
+    ).toBeAnObjectWith({ decoded: expect.defined(), fragment: expect.defined() })
```

Comment:
decoding tx is the most important thing that this fn does. Why don't you assert it?